### PR TITLE
Updates for metadata and new items in the data release

### PR DESCRIPTION
**Pull request by jordansread**

### CODE REVIEW

File: in_text/text_01_spatial.yml [hunk 1]
```diff
@@ -153,155 +160,553 @@ entities:
...
+      attr-def: >-
+        The kurtosis of temperature of observations. This value was not used as a metafeature for target lakes, 
+        even while appears in the data release.
+      attr-defs: This data release
+      data-min: NA
+      data-max: NA
+      data-units: NA
     -
       attr-label: zero_temp_doy
+      attr-def: >-
```
Comment (jordansread) [inline in_text/text_01_spatial.yml:658]: @jdwillard19 please check this for accuracy. You can comment here and I can update the PR with any changes.

File: in_text/text_01_spatial.yml [hunk 2]
```diff
@@ -153,155 +160,553 @@ entities:
...
+        of days where stratification (as specified by any time the difference between top and bottom temperatures exceeded 1 degree C)
+        existed. A glm_strat_perc value of 0 would mean this lake was simulated by PB0 to never be stratified, and a value of 1 
+        would mean this lake was simulated to be stratified constantly. 
+      attr-defs: See Hipsey et al. 2019 for details on GLM. 
+      data-min: 0
+      data-max: 0.949
+      data-units: NA
     -
       attr-label: ws_sp_mix
+      attr-def: Average windspeed during the period of spring mixing (spring unstratified period).
```
Comment (jordansread) [inline in_text/text_01_spatial.yml]: @jdwillard19 please check this for accuracy

File: in_text/text_01_spatial.yml [hunk 3]
```diff
@@ -153,155 +160,553 @@ entities:
...
+        The average day of year (DOY) when air temperatures crossed into below zero conditions. DOY is between 1 and 366, with 
+        1 representing January first.
+      attr-defs: This data release
+      data-min: 310
+      data-max: 349
+      data-units: NA
     -
       attr-label: at_amp
+      attr-def: >-
+        The estimated average seasonal amplitute of air temperature. Is a low to high range in degrees C.
```
Comment (jordansread) [inline in_text/text_01_spatial.yml:669]: @jdwillard19 please check this too

**Comment by jdwillard19** [comment]
@jread-usgs  For _Zero_temp_doy,_ first the average air temperature per day of year is calculated, then the value is calculated as the first "day of year" index of the first five days in a row of on average subzero temperatures. Then, _ws_sp_mix_  is the average windspeed for 45 days after _zero_temp_doy_, given a sequence of average windspeeds per day of year. The amplitude looks correct to me

**Comment by jordansread** [comment]
fixes #11

File: in_text/text_04_inputs.yml [hunk 4]
```diff
@@ -33,23 +45,221 @@ entities:
...
+    -
+      attr-label: perc_diff_sqrt_surface_area
+      attr-def: Percent difference in square root of surface area (absolute value of source minus target, divided by mean of source and target, and multiplied by 100).
+      attr-defs: Willard et al. 2021
+      data-min: NA
+      data-max: NA
+      data-units: NA
+    -
+      attr-label: diff_lathrop_strat
+      attr-def: Difference in surface area (source minus target). lathrop_strat is binary, so this value can only be -1, 0, or 1.
```
Comment (jdwillard19) [inline in_text/text_04_inputs.yml:212]: I think this is a typo

Comment (jdwillard19) [inline in_text/text_04_inputs.yml]: I think this surface area text is a typo

**Pull request merged**